README.md

```diff
@@ -20,6 +20,8 @@ The distributed nature of podcasting makes linking to a show on a given platform
 * Goodpods: `https://www.goodpods.com/podcasts-aid/${appleID}`
 * Google Podcasts: `https://podcasts.google.com/?feed=${base64url(feedUrl)}` or `https://podcasts.google.com/subscribe-by-rss-feed?feed=${base64url(feedUrl)}`
 * gpodder: `http://gpodder.net/subscribe?url=${feedUrl}`
+* Jam: `https://www.listentojam.com/itunes/${appleID}`
+* Listen Notes: `https://listennotes.com/itunes/id${appleID}`
```

Comment:
```suggestion
```
Listen Notes is already present in the [Resource Links](https://github.com/nathangathright/podcast-platform-links#resource-links) section.